hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleUtility.java

```diff
@@ -318,8 +312,8 @@ public static String calculateStringHash(final @NonNull Schedule scheduleToHash)
         if (scheduleToHash.scheduledTransaction() != null) {
             addToHash(hasher, scheduleToHash.scheduledTransaction());
         }
-        if (scheduleToHash.expirationTimeProvided() != null) {
-            addToHash(hasher, scheduleToHash.expirationTimeProvided());
+        if (scheduleToHash.providedExpirationSeconds() != Schedule.DEFAULT.providedExpirationSeconds()) {
+            addToHash(hasher, scheduleToHash.providedExpirationSeconds());
```

Comment:
With the change to long, instead of serializing a ProtoLong (which isn't how this is stored), we can just use `hasher.putLong(scheduleToHash.providedExpirationSeconds())`.

Reply:
Per discussion with Joseph, I added a @todo tag and he will fix this in a separate PR that he's working on.